app/services/forest_liana/search_query_builder.rb

```diff
@@ -146,6 +146,63 @@ def search_param
       @records
     end
 
+    def association_table_name(name)
+      QueryHelper.get_tables_associated_to_relations_name(@records).detect { |key, values| 
+        break key if Array(values).include?(name) 
+      }
+
+    end
+
+    def sort_query
```

Comment:
Good to factorize stuff but...
It does not look really consistent to me to move the sorting logic into a service named `search_query_builder`

Reply:
The service does the same thing that query_builder on ForestAdmin/forest-express-sequelize.
Should I rename it so that the two match together ?

Reply:
If your implementation is consistent with `forest-express-sequelize` keep it like this and forget my comment. ;)